### ISSUE TITLE
Name/Texture Customization

### ISSUE THREAD

**Issue by Sirmentio** [NathanPB/DeepMobLearning-Refabricated#27]
Hello. I've wanted to make a little modpack for my friends with this mod based around fantasy themes and the like, and while I do like the theme around glitches, but I'd like to be able to have an alternative set of names/textures themed around fantasy somehow. I'm not really to knowledgeable on how to do it via resource packs and I feel it'd be a bit of a mess that my friends themselves would have to keep track of, and I wouldn't want to do that to them.

**Comment by NathanPB** [comment]
Hey, I'll end up not doing this because of two points:

- I'm TERRIBLE with art. Most of the textures of this mod are being used from the original mod (**with** the author's permission) and I'm in need of hiring someone to redo all the textures of the mod.

- At least for now, I don't think that theme will fit nicely in the mod, tho I got no time to work in a "side" project of creating that resource pack you asked for.

However, the project is completely open source, you (everyone) are free to use the base project for anything you want ([as long as it fits in the LGPL v3 license](https://github.com/NathanPB/DeepMobLearning-Refabricated/blob/master/COPYING)), and doing a resource pack for tweaking [the translations](https://minecraft.gamepedia.com/Tutorials/Creating_a_resource_pack#Adding_Languages) and [the textures](https://minecraft.gamepedia.com/Tutorials/Creating_a_resource_pack#Modifying_an_Entity.27s_texture) is very easy (as long as you already have the fantasy textures, which is not my case).

I'll be closing this issue with a wontfix, but if someone shows up with a resourcepack about that I'll be happy in supporting the project =)

Btw, I attached two links above that can help you in tweaking item names and textures, if you are interested in proceeding with that
